### DISCUSSION TITLE
Set default padding/margin to 0 for picker/switch

### DISCUSSION THREAD

**Discussion by BlueRaja** [dotnet/maui#21020] [Ideas]
### Description

The majority of controls in MAUI have no padding/margin by default, which is great.  However `Picker`, `Switch`, and probably others do.  Even worse, these controls have no writable `Padding` property, meaning it's impossible to remove the excess whitespace without a native handler _(and in fact I [haven't been able to remove it](https://stackoverflow.com/questions/78108513/how-to-remove-native-padding-from-picker-switch) even with a native handler)_.

Ideally `Padding`/`Margin` properties would be added to the MAUI controls. I assume this hasn't been done yet because it's too hard. Fine, whatever.  But at the _very least_, please set the default padding/margin to 0.  It's very easy to _add_ padding to a control that doesn't support it, by surrounding it in a `Box` or something.  But it's nearly impossible to remove padding from a control.

### Public API Changes

n/a

### Intended Use-Case

This would be useful for basically everyone

**Comment by bradencohen** [comment]
+1. Same for RadioButton. Had to swim very upstream to get that control to look like my others.
